cli/args/flags.rs

```diff
@@ -906,6 +907,20 @@ fn compile_subcommand<'a>() -> Command<'a> {
   runtime_args(Command::new("compile"), true, false)
     .trailing_var_arg(true)
     .arg(script_arg().required(true))
+    .arg(
+      Arg::new("side-module")
+        .long("side-module")
+        .help("Additional module to include in the module graph")
```

Comment:
Let's add a note that it's a new an unstable function
```suggestion
        .help("UNSTABLE: Additional module to include in the module graph")
```